src/mos.c

```diff
@@ -91,13 +97,16 @@ static t_mosCommand mosCommands[] = {
 	{ "CREDITS",	&mos_cmdCREDITS,	NULL,			HELP_CREDITS },
 	{ "DELETE",		&mos_cmdDEL,		HELP_DELETE_ARGS,	HELP_DELETE },
 	{ "DIR",		&mos_cmdDIR,		HELP_CAT_ARGS,		HELP_CAT },
+	{ "DISC",		&mos_cmdDISC,		NULL,		NULL },

```

Comment:
I'm not really sure I see the point of the `*DISC` command - to me it just seems to slow down SD card access.  I'm not seeing any benefit in including this, so think we'd probably be better off removing it before release.

Reply:
I'll leave in `*disc` as it's essentially there for the lolz.

as there's no help text for it, it doesn't show up in the list of available commands for `*help`.  this will let us remove it later if we want to.

one flaw to note with `*disc` is that it's a one-way switch - once you've turned it on, you're stuck with disc access at roughly 5.25" disc speeds.  the only way to reset this behaviour is to press the reset button.  for an undocumented Easter egg that's fine :grin: